cypress/support/pages/editor.pageObject.js

```diff
@@ -0,0 +1,74 @@
+import PageObject from '../PageObject';
+
+class EditorPageObject extends PageObject {
+  url = '/#/editor';
+
+  get titleField() {
+    return cy.getByDataCy('article-title');
+  }
+
+  get descriptionField() {
+    return cy.getByDataCy('article-description');
```

Comment:
Ensure that the `getByDataCy` command correctly returns the element with the data-cy attribute 'article-description'.